packages/medusa/src/loaders/index.ts

```diff
@@ -77,9 +78,11 @@ export default async ({
   // Add additional information to context of request
   expressApp.use((req: Request, res: Response, next: NextFunction) => {
     const ipAddress = requestIp.getClientIp(req) as string
-    ;(req as any).request_context = {
+    req.request_context = {
       ip_address: ipAddress,
+      cache_key: v4(),
```

Comment:
self-review: Using `v4()` is probably only for testing purposes

Reply:
I understand this is for testing, but we could think about a future use case where services need to share this `cache_key` and the value should be accusable globally.
it could be a custom header `req.headers["x-cache-key"] ?? v4()` or something like that.
Same would apply for logging using the same `x-request-id`

Reply:
suggestion: I think we should have a naming more comprehensible than just cache key. I think event should be part of the name as it is specific to the event. We probably don't want someone trying to store something on it that is very unrelated because of a missunderstanding